Sources/Private/Utility/Helpers/View+ValueChanged.swift

```diff
@@ -8,7 +8,7 @@ import SwiftUI
 extension View {
   /// A backwards compatible wrapper for iOS 14 `onChange`
   @ViewBuilder
-  func valueChanged<T: Equatable>(value: T, onChange: @escaping (T) -> Void) -> some View {
+  func valueChanged(value: any Equatable, onChange: @escaping (any Equatable) -> Void) -> some View {
```

Comment:
I think we'll need to use the old syntax here since Lottie still supports Swift 5.5. CI will probably fail with this written as-is.

```suggestion
  func valueChanged<T: Equatable>(value: T, onChange: @escaping (T) -> Void) -> some View {
```

Reply:
Yeah, CI is failing so we can't use `any` like this

Reply:
Oh no, @calda we could introduce a new type such as [AnyEquatable]( https://gist.github.com/JadenGeller/f0d05a4699ddd477a2c1) and still being able to migrate off `AnyHashable`. wdyt?

Reply:
Sure, that seems reasonable to me. Better than adding an unnecessary `Hashable` constraint.